src/js/views/patients/widgets/widgets.js

```diff
@@ -1,15 +1,12 @@
-import { each, propertyOf, reduce, extend, isFunction, isString, filter, reject, find } from 'underscore';
+import { extend, isFunction, find } from 'underscore';
 import Radio from 'backbone.radio';
-import { View, CollectionView } from 'marionette';
+import { View } from 'marionette';
 import dayjs from 'dayjs';
-import parsePhoneNumber from 'libphonenumber-js/min';
 
 import hbs from 'handlebars-inline-precompile';
 
 import Handlebars from 'handlebars/dist/cjs/handlebars';
 
-import patientTemplate from 'js/utils/patient-template';
```

Comment:
Just realized... I don't think this is used anywhere else in the codebase. So we should be able to delete it. Along with its component test.

Reply:
Looks like it's just testing that the handlebars templates are properly converted to HTML.